File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/TokenVo.java
package org.javamaster.invocationlab.admin.model.erd;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

import java.io.Serializable;

@Data
public class TokenVo implements Serializable {
    @JsonProperty("access_token")
    private String accessToken;
    @JsonProperty("token_type")
    private String tokenType;
    @JsonProperty("refresh_token")
    private String refreshToken;
    @JsonProperty("expires_in")
    private int expiresIn;
    private String scope;
    @JsonProperty("tenant_id")
    private String tenantId;
    private String license;
    @JsonProperty("dept_id")
    private String deptId;
    @JsonProperty("dept_name")
    private String deptName;
    @JsonProperty("user_id")
    private String userId;
    private String username;
    private String email;
    private String mobileNo;
    private String env;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/TokenVo.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

import com.fasterxml.jackson.annotation.JsonProperty
import java.io.Serializable


@AllOpen
class TokenVo : Serializable {
    @JsonProperty("access_token")
    var accessToken: String? = null

    @JsonProperty("token_type")
    var tokenType: String? = null

    @JsonProperty("refresh_token")
    var refreshToken: String? = null

    @JsonProperty("expires_in")
    var expiresIn = 0
    var scope: String? = null

    @JsonProperty("tenant_id")
    var tenantId: String? = null
    var license: String? = null

    @JsonProperty("dept_id")
    var deptId: String? = null

    @JsonProperty("dept_name")
    var deptName: String? = null

    @JsonProperty("user_id")
    var userId: String? = null
    var username: String? = null
    var email: String? = null
    var mobileNo: String? = null
    var env: String? = null
}
